apps/cadence_spectre/build_spectre.sh

```diff
@@ -16,7 +16,7 @@ install_required_packages()
         sudo yum -y install ksh
```

Comment:
Do you need to call sudo on each installation line ? can't you just call the script in sudo instead ?